trades.py

```diff
@@ -21,16 +21,20 @@
 """This script queries the OMEN subgraph to obtain the trades of a given address."""
 
 import datetime
+import re
 import time
 from argparse import ArgumentParser
 from collections import defaultdict
 from enum import Enum
+from pathlib import Path
 from string import Template
-from typing import Any
+from typing import Any, Dict, List
 
 import requests
 
+from scripts.mech_events import MechRequest, get_mech_requests
 
+IRRELEVANT_TOOLS = ["openai-text-davinci-002", "openai-text-davinci-003", "openai-gpt-3.5-turbo", "openai-gpt-4", "stabilityai-stable-diffusion-v1-5", "stabilityai-stable-diffusion-xl-beta-v2-2-2", "stabilityai-stable-diffusion-512-v2-1", "stabilityai-stable-diffusion-768-v2-1", "deepmind-optimization-strong", "deepmind-optimization"]
```

Comment:
This is stored on-chain. Should be fetched to avoid having to manually update it.

Reply:
I believe this is not stored on-chain. Probably you refer to available tools, but in this case is the tools the trader is excluding from the available ones.

@Adamantios please correct me if I'm wrong.